index.d.ts

```diff
@@ -11,7 +11,6 @@ export interface RequestContext {
 
 export type Hook =
   | 'onRequest'
-  | 'preParsing'
```

Comment:
Not sure about this one. To me it should be removed from the hooks types. WDYT ?
Note that It will continue to exist as an alias of `onRequest` but in the meantime it is in the last stage of its deprecation.

Reply:
Sounds good